### DISCUSSION TITLE
Wallpaper Randomizer

### DISCUSSION THREAD

**Discussion by t3st-account** [regolith-linux/regolith-desktop#811] [Ideas]
It would be nice if there was a built in way to specify a folder containing images instead of a sole file for the regolith.wallpaper.file atribute. This would ideally enable a random wallpaper on Login.

I am aware tha this is possible via scropt of some sort that would randomize the image, butr it would be nice if there was a builtin way

**Comment by jeromelenoir** [comment]
Hello,
Just my personal experience... I use pywal which randomizes the wallpaper and also adapt the color palette of the terminal and Firefox....

**Comment by Wylbur** [comment]
Another good option for a wallpaper changer is Variety, https://peterlevi.com/variety/
